app/decorators/contribution_decorator.rb

```diff
@@ -0,0 +1,20 @@
+class ContributionDecorator < Draper::Decorator
+  delegate_all
+
+  def user_short_name
+    user.name.split.values_at(0, -1).uniq.join(' ')
```

Comment:
this should be on the user model, or user decorator, is not concern of the contribution
so you could only use a delegate here

Reply:
That's why I prefer to think of draper decorators as presenters that's what they actually are, I mean, this rule is specific to the contributions page, isn't it?